### DISCUSSION TITLE
Moving the integrations out

### DISCUSSION THREAD

**Discussion by ckipp01** [scalacenter/bloop#1887] [General] [answered]
👋🏼 so quick question on this. So after the work that was done in https://github.com/scalacenter/bloop/pull/1868 to move `bloop-config` out there was comments about removing them in [here](https://github.com/scalacenter/bloop/pull/1868#pullrequestreview-1193028655) and also I see it's listed under https://github.com/scalacenter/bloop/issues/1680. So this discussion is just to verify what we want to do. Then I can dive in. Mainly:

- Which integrations do we remove? All of them? Just Maven and Gradle?

Once we agree on this I can start ripping them out. I figured this might be easier to do before we migrate the sbt-projectmatrix and sbt-ci-release.

**Comment by tgodzik** [comment, marked as answer]
Maybe migrating just Maven and Gradle would make sense, since they are unrelated to anything else. We could also use the actual build tools for releasing and get rid of some workarounds in sbt. Sbt is another issue since it's used for generating `.bloop` config for some of the tests. Might be good to keep it here.

**Reply by ckipp01** [reply]
> Maybe migrating just Maven and Gradle would make sense, since they are unrelated to anything else.

Sounds good. I'll start working on this.

> We could also use the actual build tools for releasing and get rid of some workarounds in sbt

Ha, I'll give it a try.
